corehq/apps/locations/tests/test_location_fixtures.py

```diff
@@ -1,16 +1,22 @@
 from django.test import SimpleTestCase
-from corehq.apps.locations.models import Location
+from corehq.apps.locations.models import SQLLocation, LocationType
 from ..fixtures import _location_to_fixture, _location_footprint
 
 
 class LocationFixturesTest(SimpleTestCase):
 
     def test_metadata(self):
-        location = Location(
-            _id="unique-id",
+        state = LocationType(
+            domain="test-domain",
+            name="state",
+            code="state",
+        )
```

Comment:
for future reference, if you make sure the location type exists first (via the `bootstrap_location_types` test util, for instance), you can still create a couch `Location` with the string representation of the location type.  It looks up the location type object on save.